Pod/Classes/GraphQL/ARGraphQLQueryCache.m

```diff
@@ -1,121 +1,37 @@
 
 static NSURL *_cacheDirectory = nil;
 
-+ (void)load;
-{
-    // RN ceremony
-    RCT_EXTERN void RCTRegisterModule(Class);
-    RCTRegisterModule(self);
-
-    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
-    _cacheDirectory = [NSURL fileURLWithPath:[paths[0] stringByAppendingPathComponent:@"RelayResponseCache"]];
-
-    // Ensure the cache directory exists
-    [[NSFileManager defaultManager] createDirectoryAtURL:_cacheDirectory
-                             withIntermediateDirectories:YES
-                                              attributes:nil
-                                                   error:nil];
-
-    dispatch_async(dispatch_get_global_queue(QOS_CLASS_UTILITY, 0), ^{
-        NSError *error = nil;
-
-        NSURL *preHeatedCacheDirectory = [[NSBundle mainBundle] URLForResource:@"PreHeatedGraphQLCache" withExtension:nil];
-        NSArray<NSURL *> *preHeatedCacheEntries = ReadCacheEntries(preHeatedCacheDirectory, &error);
-        NSArray<NSURL *> *existingCacheEntries = error ? nil : ReadCacheEntries(_cacheDirectory, &error);
-        if (error) {
-            NSLog(@"[ARGraphQLQueryCache] Error while loading pre-heated GraphQL cache: %@", error);
-            return;
-        }
-
-        NSArray<NSString *> *existingCacheEntryKeys = [existingCacheEntries valueForKey:@"lastPathComponent"];
-
-        for (NSURL *cacheEntryURL in preHeatedCacheEntries) {
-            error = nil;
-
-            NSDictionary *cacheEntry = ReadCacheEntry(cacheEntryURL, &error);
-            if (error) {
-                NSLog(@"[ARGraphQLQueryCache] Error while loading pre-heated GraphQL cache entry `%@': %@", cacheEntryURL.path, error);
-                continue;
-            }
-
-            NSNumber *freshness = cacheEntry[@"freshness"];
-            NSParameterAssert(freshness);
-            // Skip pre-heated cache that is considered out-of-date.
-            if (!IsCacheValid([NSDate dateWithTimeIntervalSince1970:freshness.unsignedIntegerValue])) {
-                DLog(@"[ARGraphQLQueryCache] Stale pre-heated GraphQL cache entry will be ignored: %@", cacheEntryURL.path);
-                continue;
-            }
-
-            NSDictionary *queryParams = cacheEntry[@"queryParams"];
-            NSParameterAssert(queryParams);
-            NSString *queryID = queryParams[@"documentID"];
-            NSParameterAssert(queryID);
-            NSDictionary *variables = queryParams[@"variables"];
-            NSParameterAssert(variables);
-            NSString *cacheKey = CacheKey(queryID, variables);
-
-            // Skip pre-heated cache entries that already exist.
-            if ([existingCacheEntryKeys indexOfObject:cacheKey] != NSNotFound) {
-                DLog(@"[ARGraphQLQueryCache] Existing pre-heated GraphQL cache entry will be ignored: %@", cacheEntryURL.path);
-                continue;
-            }
```

Comment:
So this was the bug, the entry is deleted when the cache is invalidated (on mutation) and then on next launch would put it back into place again.